iris/meta.yaml

```diff
@@ -20,14 +20,13 @@ requirements:
         - biggus >=0.14
         - cartopy >=0.14
         - netcdf4
-        - numpy x.x
+        - numpy
```

Comment:
This (`numpy`) is not the same as `numpy x.x`. Are you intentionally adjusting the numpy requirement here?